src/ImageCropBundle/Bridge/Chameleon/Field/TCMSFieldMediaWithImageCrop.php

```diff
@@ -54,6 +56,12 @@ public function ChangeFieldDefinition($sOldName, $sNewName, &$postData = null)
 
         $logChangeDataModels[] = new LogChangeDataModel($query);
         TCMSLogChange::WriteTransaction($logChangeDataModels);
+
+        $translatedFieldDefinition = clone $this->oDefinition;
```

Comment:
What happens here if the field is created (first change)?

Reply:
The logic in UpdateFieldTranslationKeys() works in every case and is called for the main field (and for any other field type) in this way, too.